js/sign/src/signers/integrity-block-signer.ts

```diff
@@ -134,17 +130,11 @@ export class IntegrityBlockSigner {
     return new Uint8Array(buffer);
   }
 
-  signAndVerify(
+  verifySignature(
     dataToBeSigned: Uint8Array,
```

Comment:
nit: Rename to just `data`

Reply:
I will do this in a separate PR as this naming convention is used in several places over the code and it's not really a change part of this PR.